src/Agent.PluginHost/Agent.PluginHost.csproj

```diff
@@ -6,6 +6,11 @@
     <OutputType>Exe</OutputType>
   </PropertyGroup>
 
+  <PropertyGroup>
```

Comment:
can this be put in the Common.props?

Reply:
Testing now to see if it works since the nesting levels are different across projects and the ruleset is referenced as a relative path.